src/js/themes/base.js

```diff
@@ -1060,6 +1060,9 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         // any text props
         color: 'text',
         weight: 'bold',
+        margin: {
```

Comment:
It seems a little hidden to me that this is only valid when in a column context. As the theme is structured, I would expect this to be applied all of the time. Given how we are using this. I'm wondering if we should instead place this under something like `theme.nameValuePair.column.gap` or `theme.nameValuePair.column.name.margin`. I have a slight preference for the former, as I think it aligns better with `theme.nameValueList.gap`.

Reply:
good point I made it so that within the `NameValuePair.js` this is under the `name` 
```
  margin={column ? theme.nameValuePair.margin : 'undefined'}

```
which by looking at the theme this does not be as appropriate. I originally leaned towards `theme.nameValuePair.column.gap` however since this is only acting like a `gap` and really a  `margin` under the hood I think something like `theme.nameValuePair.column.name.margin` would be appropriate